TypeScript Error:  Cannot find module 'node:stream' or its corresponding type declarations.

### @inertiajs/vue3 Version

2.3.17

### Backend stack (optional)

_No response_

### Describe the problem

Checking my project's TypeScript syntax with `vue-tsc --noEmit` gives the following error:

```
node_modules/@vue/server-renderer/dist/server-renderer.d.ts:2:36 - error TS2307: Cannot find module 'node:stream' or its corresponding type declarations.

2 import { Writable, Readable } from 'node:stream';
```

This is due to PR #2648 adding `import { renderToString } from 'vue/server-renderer'` to `vue3/src/createInertiaApp.ts`. 

Importing `vue/server-renderer` in turn tries to import `node:stream`, which fails because it is not in a NodeJS environment, it is in a web browser environment.

Adding `@types/node` to the project is not a viable solution because it conflicts with client side definitions (for example `setTimeout(): number` vs `setTimeout(): Timeout`).

### Steps to reproduce

Create a file `app.ts` containing:

```ts
import { createInertiaApp } from '@inertiajs/vue3';
```

Run:

`npx vue-tsc --noEmit`

--- a/packages/vue3/src/createInertiaApp.ts
+++ b/packages/vue3/src/createInertiaApp.ts
@@ -11,7 +11,6 @@ import {
   SharedPageProps,
 } from '@inertiajs/core'
 import { createSSRApp, DefineComponent, h, Plugin, App as VueApp } from 'vue'
-import { renderToString } from 'vue/server-renderer'
 import App, { InertiaApp, InertiaAppProps, plugin } from './app'
 import { config } from './index'
 import { VueInertiaAppConfig } from './types'
@@ -40,7 +39,7 @@ type InertiaAppOptionsForSSR<SharedProps extends PageProps> = CreateInertiaAppOp
   VueApp,
   VueInertiaAppConfig
 > & {
-  render: typeof renderToString
+  render: (app: VueApp) => Promise<string>
 }
 
 export default async function createInertiaApp<SharedProps extends PageProps = PageProps & SharedPageProps>(

